config/local/multi-tenancy/domain/api-catalog-service.yml

```diff
@@ -0,0 +1,73 @@
+spring.profiles.include: diag
```

Comment:
you don't need to have another file with identical configuration properties, you can pass multiple files via 
spring.config.additional-location=file:./config/local/api-catalog-service.yml,file:./config/local/multi-tenancy/domain/api-catalog-service.yml and set only those that are different

Reply:
Good point, I'll address it, thanks